### PR TITLE
Added watches for when radio buttons change and the ability to set an aria label

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
#27 and #28 are addressed.

### CODE REVIEW

File: src/app/public/modules/radio/radio-group.component.ts [hunk 1]
```diff
@@ -87,16 +90,18 @@ export class SkyRadioGroupComponent implements AfterContentInit, ControlValueAcc
...
+    this.resetRadioButtons();
 
     // Watch for radio selections.
     this.radios.forEach((radio) => {
       radio.change.subscribe((change: SkyRadioChange) => {
         this.writeValue(change.value);
       });
     });
+
+    this.radios.changes.subscribe(() => {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/radio/radio-group.component.ts]: Do we need a `.takeUntil()` on this subscribe?
I also noticed there's a leftover `.subscribe()` from before on line `97`.

**Comment by blackbaud-johnly** [comment]
I created https://github.com/blackbaud/skyux2-docs/issues/356 to document this and have the docs set up in https://github.com/blackbaud/skyux2-docs/pull/357.

File: src/app/public/modules/radio/radio-group.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/radio/radio-group.component.ts]: Fixed

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/radio/radio-group.component.ts]: As far as "there's a leftover `.subscribe()` from before on line `97`" I'm not sure what you mean there. That is for different functionality. I did add the `.takeUntil()` there too if that is what you meant.

**Pull request merged**